int32 mode
float32 time
float32[3] pos
---
string status
